src/connector/src/sink/big_query.rs

```diff
@@ -678,49 +773,39 @@ impl StorageWriterClient {
         )
         .await
         .map_err(|e| SinkError::BigQuery(e.into()))?;
-        let client = conn.conn();
-        Ok(StorageWriterClient {
-            client,
-            environment,
-        })
+        let mut client = conn.conn();
+
+        let (tx, rx) = mpsc::channel(BIGQUERY_SEND_FUTURE_BUFFER_MAX_SIZE);
```

Comment:
We can use bounded channel here since we already limit the queue size in the `select!`